packages/test/test-gc-sweep-tests/src/dataObjectWithCounter.ts

```diff
@@ -31,7 +32,7 @@ export class DataObjectWithCounter extends DataObject {
         this.counter = await handle.get();
     }
 
-    public async sendOp() {
+    protected sendOp() {
```

Comment:
This function is unncessary as its just one interesting line. It can be done inside `sendOps` directly.

Reply:
I think it can be interesting if you see my comment about refactoring the loop logic.  Probably rename it to `runStep`

Reply:
I'm going to remove it, and then we can add it back if we need the separation of logic